openedx/core/djangoapps/programs/__init__.py

```diff
@@ -8,3 +8,9 @@
 To ensure maximum separation of concerns, and a minimum of interdependencies,
 this package should be kept small, thin, and stateless.
 """
+from openedx.core.djangoapps.waffle_utils import (WaffleSwitch, WaffleSwitchNamespace)
+
+PROGRAMS_UTILS_WAFFLE_SWITCH_NAMESPACE = WaffleSwitchNamespace(name='programs_utils')
+
+ALWAYS_CALCULATE_PROGRAM_PRICE_AS_ANONYMOUS_USER = WaffleSwitch(
```

Comment:
Pleas add a comment mentioning this is meant to be temporary (assuming it is), with a link to clean-up JIRA ticket.